CharacterCreation/Class-Specific Documentation/Doctor/Doctor Malpractice Procedures.txt

```diff
@@ -496,5 +495,5 @@ You gain 1 Contagion.
 Fortitude is permanently reduced by 2. You gain 2 Contagions.
 
     Their Lymph Node: The target loses their lymph node. They lose 40 health and
-up to 120 Nanites. This ability will not drop a target's nanite count below 0. 
+cannot use nanites. This ability will not drop a target's nanite count below 0. 
```

Comment:
Excellent conversion. Still need to remove the next sentence about nanite count.
Also suggest cannot use ^medical^ nanites to leave room for nanite-based items and distinguish from the Suppressed condition.